aikau/src/main/resources/alfresco/forms/Form.js

```diff
@@ -515,7 +585,7 @@ define(["dojo/_base/declare",
        * @instance
        */
       createButtons: function alfresco_forms_Form__createButtons() {
-         if (this.showOkButton === true)
+         if (this.showOkButton === true && this.autoSavePublishTopic === null)
```

Comment:
Confused by this... why not !this.autoSavePublishTopic... having the empty string as a valid topic wouldn't make a lot of sense.